bin/check-windows-disk.rb

```diff
@@ -31,6 +31,10 @@ class CheckDisk < Sensu::Plugin::Check::CLI
   option :fstype,
          short: '-t TYPE',
          proc: proc { |a| a.split(',') }
+ 
+  option :onlymnt,
```

Comment:
I think a better name would be `mount_points`

Reply:
Changed